kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/parquet/ParquetFileReader.java

```diff
@@ -99,31 +98,67 @@ public ColumnarBatch next() {
                 do {
                     hasNotConsumedNextElement = false;
                     // hasNext reads to row to confirm there is a next element.
+                    // get the row index only if required by the read schema
+                    long rowIndex = hasRowIndexCol ? reader.getCurrentRowIndex() : -1;
+                    readSupport.finalizeCurrentRow(rowIndex);
```

Comment:
Can you update the javadoc for `finalizeCurrentRow` (I think it's here: https://github.com/vkorukanti/delta/blob/1a94f3f0434dfc8f7b50518ef8cb090ad5c21365/kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/parquet/ParquetConverters.java#L118)

to explain what to do when it receives a `-1`?

Reply:
`finalizeCurrentRow` doesn't have any special behavior when the value is -1. It just populates the given value if there is a row_index column being read. I can add the doc, but it is not going to add any value.